tests/scripts/gen_release_notes.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+set -e
+
+function help() {
+  print="
+  suppose we want release note for v1.6.8, we'll check the list of pr merged in upstream/release-1.6 but not present in v1.6.7
+  EXAMPLE command:
+  ./tests/scripts/gen_release_notes.sh <rook/rook remote name>/release-1.6 v1.6.7
+  "
+  echo "$print"
+  exit 1
+}
+
+FROM_BRANCH=$1 # example branch name, "upstream/release-1.6"
+TO_TAG=$2 # example tag, "v1.6.7"
+
+if [[ $# -ne 2 ]]; then
+  echo "requires exactly 2 arguments"
+  help
+fi
+
+if ! git remote show $(echo $1 | cut -d / -f1)|grep -oh $(echo $1 | cut -d / -f2-) >/dev/null 2>&1 ; then
+  echo "remote doesn't exist"
+  exit 1
+fi
+
+if ! git show-ref --tags $2 -q; then
+  echo "tag doesn't exists"
```

Comment:
```suggestion
  echo "tag doesn't exist"
```

ditto print tag name